cmd/bootkube/render.go

```diff
@@ -24,16 +24,20 @@ var (
 	}
 
 	renderOpts struct {
-		assetDir    string
-		etcdServers string
-		apiServers  string
-		altNames    string
+		assetDir       string
+		caFile         string
+		privateKeyFile string
+		etcdServers    string
+		apiServers     string
+		altNames       string
 	}
 )
 
 func init() {
 	cmdRoot.AddCommand(cmdRender)
 	cmdRender.Flags().StringVar(&renderOpts.assetDir, "asset-dir", "", "Output path for rendered assets")
+	cmdRender.Flags().StringVar(&renderOpts.caFile, "ca-file", "", "Path to an existing PEM encoded CA.")
```

Comment:
s/ca-file/ca-certificate

Also in the help text, maybe add some behavior notes:

"If set, TLS assets will be generated using this certificate authority"